modules/core/src/tools/file/vpIoTools.cpp

```diff
@@ -1705,6 +1705,7 @@ std::vector<std::string> vpIoTools::getDirFiles(const std::string &pathname) {
     free(list[i]);
   }
   free(list);
+  std::sort(files.begin(), files.end());
   return files;
```

Comment:
Thanks, but could you also add the `std::sort` line for Windows case (near line 1734)

Reply:
Yes, it's done in next commit